server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -212,6 +212,23 @@ func (e *EventChannel) HandleEvent(ctx context.Context, event *pepb.PublishBuild
 		}
 	}
 
+	// When we get the workspace status event, update the invocation in the DB
+	// so that it can be searched by its commit SHA, user name, etc. even
+	// while the invocation is still in progress.
+	if isWorkspaceStatusEvent(bazelBuildEvent) {
+		db := e.env.GetInvocationDB()
+		ti, err := db.LookupInvocation(ctx, iid)
+		if err != nil {
+			return err
+		}
+		proto := TableInvocationToProto(ti)
```

Comment:
I think this probably works fine but usually it's better to not shadow a name of a library (in this case the "proto" library). Maybe call it tp? or invocationProto?

Reply:
Lol, I was too lazy to scroll up and see if proto was imported already so I was relying on the compiler to tell me there was an error. Good to know that Go doesn't report errors in this case